app/code/Magento/InventoryApi/Api/Data/ReservationInterface.php

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\InventoryApi\Api\Data;
+
+use Magento\Framework\Api\ExtensibleDataInterface;
+use Magento\InventoryApi\Api\Data\ReservationExtensionInterface;
+
+/**
+ * The entity responsible for reservations, created to keep inventory amount (product quantity) up-to-date.
+ * It is created to have a state between order creation and inventory deduction (deduction of specific SourceItems)
+ *
+ * @api
+ */
+interface ReservationInterface extends ExtensibleDataInterface
+{
+    /**
+     * Constants for keys of data array. Identical to the name of the getter in snake case
+     */
+    const RESERVATION_ID = 'reservation_id';
+    const STOCK_ID = 'stock_id';
+    const SKU = 'sku';
+    const QUANTITY = 'quantity';
+    const STATUS = 'status';
+
+    /**#@+
+     * Reservation possible statuses. Maybe make sense to introduce extension point for Reservation Open-Close statuses
+     */
+    const STATUS_ORDER_CREATED = 1; // For Order Placed
```

Comment:
@aleron75  let's leave it as-is now. When all constants are in this class.
And will discuss  how it can be improved later. 
Because for now, we don't have a clear vision how this should look like